lib/licenses.nix

```diff
@@ -1212,6 +1212,12 @@ lib.mapAttrs mkLicense ({
     fullName = "TCL/TK License";
   };
 
+  tost = {
+    shortName = "TSL";
+    fullName = "Tomorrow Open Source License";
```

Comment:
```suggestion
    fullName = "Tomorrow Open Source Technology License 1.0";
```

Reply:
Thats not the official name

Reply:
It's what it says right in the license text.

Reply:
> To date, USD has been distributed under what we called a modified Apache 2.0 License. We now have a new name for our license: the “Tomorrow Open Source License” or “TOST.”

as it is written on there blog. But I agree with you that we should name it after their official LICENSE text

Reply:
It's also good to include the version at least here in the fullName as they may have a v2 at some point.